.github/workflows/build.yml

```diff
@@ -62,6 +63,9 @@ jobs:
         yarn install
         yarn run test
 
+    - name: Run prettyfier
+      run: yarn run prettier --check .
```

Comment:
```
run: |
  cd js
  yarn run prettier --check .
```

Reply:
Ah, I thought it'd pick it up from the command above.
Thanks!

Reply:
I recall there was a `working-directory` option that you can use otherwise. Something like:
```
run: yarn run prettier --check .
working-directory: js
```